### ISSUE TITLE
Missing AMD64-WINDOWS\libtensorrt_llm_batch_manager_static.pre_cxx11.a

### ISSUE THREAD

**Issue by andyluo7** [NVIDIA/TensorRT-LLM#179]
I  tried to build tensort-llm from source on windows 11. The following error occurred. 

  Building Custom Rule C:/Users/amd/inference/TensorRT-LLM/cpp/tensorrt_llm/CMakeLists.txt
  Auto build dll exports
LINK : fatal error LNK1181: cannot open input file 'C:\Users\amd\inference\TensorRT-LLM\cpp\tensorrt_llm\batch_manager\
AMD64-WINDOWS\libtensorrt_llm_batch_manager_static.pre_cxx11.a' [C:\Users\amd\inference\TensorRT-LLM\cpp\build\tensorrt
_llm\tensorrt_llm.vcxproj]
Traceback (most recent call last):
  File "C:\Users\amd\inference\TensorRT-LLM\scripts\build_wheel.py", line 248, in <module>
    main(**vars(args))
  File "C:\Users\amd\inference\TensorRT-LLM\scripts\build_wheel.py", line 152, in main
    build_run(
  File "C:\Users\amd\anaconda3\envs\trt-llm\lib\subprocess.py", line 526, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command 'cmake --build . --config Release --parallel 16 --target tensorrt_llm tensorrt_llm_static nvinfer_plugin_tensorrt_llm th_common ' returned non-zero exit status 1.

I checked the files. It has Linux library only. Please let me know how to get AMD64-WINDOWS\libtensorrt_llm_batch_manager_static.pre_cxx11.a. 

![image](https://github.com/NVIDIA/TensorRT-LLM/assets/43718156/c8e45863-32ac-425c-8689-7804ac8723c2)

**Comment by jdemouth-nvidia** [comment]
Hi @andyluo7 ,

The library for Windows is missing. We’ll fix that soon (Monday or Tuesday). Sorry for the inconvenience. 

Thanks,
Julien

**Comment by andyluo7** [comment]
@jdemouth-nvidia , has it been fixed?

**Comment by juney-nvidia** [comment]
> @jdemouth-nvidia , has it been fixed?

The engineers supporting TensorRT-LLM Windows are still working on this and it takes some time to make the release.
If you are in hurry, you can cherry-pick this [MR](https://github.com/NVIDIA/TensorRT-LLM/pull/221/files#diff-1f6a608710ec73dc4eef2ae372994e0d09bdefb889e9864cdf825d2c69c6bc10) firstly to see whether it works in your environment to unblock your dependency.  To be noted that this MR will not get merged into github directly since our current process is that MRs will be merged into internal repo firstly, then go through the validation, after finishing the validation the code will be published to github. 

Thanks
June

**Comment by juney-nvidia** [comment]
Closed as it has already been fixed in both main and release branches.

**Comment by Sonicwanjie** [comment]
0.71编译不通过，能提供新版本的window的文件吗
